dev/README_RELEASE_AIRFLOW.md

```diff
@@ -852,7 +857,7 @@ Documentation for providers can be found in the ``/docs/apache-airflow`` directo
 
     ```shell script
     cd "${AIRFLOW_REPO_ROOT}"
-    breeze build-docs apache-airflow docker-stack --clean-build
+    breeze build-docs --package-filter apache-airflow --package-filter docker-stack --clean-build
```

Comment:
```suggestion
    breeze build-docs --package-filter apache-airflow --package-filter docker-stack --clean-build
```

No need to add `--package-filter` the original version should work (and even be auto-completable). The `--package-filter` is something of a past now that most people should not use (including release managers).